CashOverflow/Models/Companies/Exceptions/LockedCompanyException.cs

```diff
@@ -0,0 +1,17 @@
+﻿// --------------------------------------------------------
+// Copyright (c) Coalition of Good-Hearted Engineers
+// Developed by CashOverflow Team
+// --------------------------------------------------------
+
+using System;
+using Xeptions;
+
+namespace CashOverflow.Models.Companies.Exceptions
+{
+    public class LockedCompanyException : Xeption
+    {
+        public LockedCompanyException(Exception innerException) 
+            : base(message: "Company is locked, please try again", innerException)
```

Comment:
remove please and add `later`. So, if the object is locked, they should give some time

Reply:
Also add dot (.)